openedx/features/smart_referral/tasks.py

```diff
@@ -0,0 +1,17 @@
+from celery.task import task
+from django.conf import settings
+
+from common.lib.mandrill_client.client import MandrillClient
+
+
+@task(routing_key=settings.HIGH_PRIORITY_QUEUE)
+def task_referral_and_toolkit_emails(contact_emails, user_email):
```

Comment:
can we please rename to `task_send_referral_and_toolkit_emails` just to be more specific